genstudio-create-validation/src/genstudiopem/web-src/src/components/RightPanel.tsx

```diff
@@ -55,66 +67,87 @@ export default function RightPanel(): JSX.Element {
   return (
     <Provider theme={defaultTheme}>
       <View backgroundColor="static-white" height="100vh">
-        <Flex direction="column" height="100%">
-          {experiences && experiences.length > 0 ? (
-            <Flex direction="column" gap="size-200">
-              <View paddingX="size-200" paddingY="size-100">
-                <ComboBox 
-                  label="Select Experience to Run Claims Check" 
-                  align="start"
-                  onSelectionChange={(key: React.Key | null) => { 
-                    if (key !== null) {
-                      const index = experiences.findIndex(exp => exp.id === key);
-                      if (index !== -1) {
-                        setSelectedExperienceIndex(index);
-                      }
-                    }
-                  }}
-                >
-                  {experiences.map((experience, index) => (
-                    <Item key={experience.id}>{`Experience ${index + 1}`}</Item>
-                  ))}
-                </ComboBox>
-              </View>
-
-              {selectedExperienceIndex !== null && (
+        {isLoading ? (
+          <Spinner message="Loading experiences..." />
```

Comment:
this one does the spinner slightly differently. it blocks the whole screen and is the regular spinner. i wasn't sure which way we prefer